plugins/languages/en/admin_dictionary.php

```diff
@@ -81,6 +81,8 @@
     'admin.fman.selected.move.prompt' => 'Enter the new location (relative to the current directory)',
     'admin.fman.selected.delete' => 'Remove',
     'admin.fman.selected.delete.confirm' => 'Do you really want to delete the selected files?',
+    'admin.fman.selected.download' => 'Download (ZIP)',
+    'admin.fman.selected.download.confirm' => 'Do you really want to download the selected files?',
```

Comment:
I'd ommit `really` since this is not a destructive action.

`Do you want to download the selected files?`

Same in czech: `Chcete stáhnout zaškrtnuté soubory?'`

Reply:
Yes, I was thinking of a case where a potentially large temp file could be created.  So I thought it appropriate to ask first.

Reply:
Done